pkg/apis/psmdb/v1/perconaservermongodbbackup_types.go

```diff
@@ -13,9 +13,9 @@ import (
 type PerconaServerMongoDBBackupSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	PSMDBCluster string              `json:"psmdbCluster,omitempty"`
-	StorageName  string              `json:"storageName,omitempty"`
-	Comperssion  pbm.CompressionType `json:"compressionType,omitempty"`
+	ClusterName string              `json:"clusterName,omitempty"`
```

Comment:
I think we should keep backward compatibility.
let's check with @spron-in

Reply:
I think we should! @spron-in please check.

Reply:
Yeah, we should. I left a comment in the JIRA ticket:

We are going to use `clusterName` everywhere

In 1.12 - 1.15 we are going to allow users to use `psmdbCluster` and `clusterName` in backup.yaml. But after 1.15 we are going to remove psmdbCluster completely.